### DISCUSSION TITLE
Draging / resizing on touch screens?

### DISCUSSION THREAD

**Discussion by ajtompkins** [vkurko/calendar#291] [General]
I've got an implementation of the calendar working well. Currently trying to make it work with the mobile friendly version of the website. 

Is there anyway of being able to drag events or resize events with a touch screen device? It either seems to register as a click or scrolls the entire screen rather than drag the event. Can't 'select' the bottom of the event box to make the event longer at all. 

It seems to work a little better if I set a 'height' property for the calendar, but still not ideal. 

Just wondering if anyone's got the calendar working nicely on touch screen devices and if I'm missing a trick?

**Comment by vkurko** [comment]
The trick is that on a mobile device you need to tap on the event and hold it for a while. After that, it becomes draggable and works like on a desktop. How long you need to hold it is configured by [longPressDelay](https://github.com/vkurko/calendar?tab=readme-ov-file#longpressdelay) option.

**Comment by ajtompkins** [comment]
That works a treat! Thanks for all the work on this projet, much appreciated.
